src/main/java/org/candlepin/subscriptions/product/OfferingSyncController.java

```diff
@@ -64,6 +94,20 @@ public void syncOffering(Offering newState) {
     // taskManager.reconcileCapacityForOffering(...); //NOSONAR
   }
 
+  private void sortOidsAndSkus(Optional<Offering> oOffering) {
```

Comment:
Lombok is used for the Offering impl, and uses whatever Lombok creates for the equals and hashcode impls. I didn't want to erronously update an Offering when the only difference was in the ordering of the items in the childSkus and productIds lists.

If we want to do something else other than normalizing these lists by sorting them, let me know.

Reply:
That was my preference as well but wasn't sure if there was a reason we wanted List. I will update Offering to use Set.